util/config/config.go

```diff
@@ -66,9 +66,11 @@ const (
 )
 
 const (
-	defaultConfigFile          = "config.json"
-	defaultSyncMemoryPercent   = 10
-	defaultSyncBatchWindowSize = 64
+	defaultConfigFile             = "config.json"
+	defaultSyncMaxMemoryPercent   = 25
+	defaultSyncBatchWindowSize    = 64
+	defaultTxPoolMaxMemoryPercent = 0.4
```

Comment:
4MB TxPoolMaxMemory is too many headcount for txPool since minimal size transaction just only 170~180 Bytes per txn.
4MB/180B ≈23.3K txn in txPool. It is costly for each getLength of sync.Map.

Reply:
The current value is 32M :)

4MB should be fine if we can eliminate memory leak. Currently there is no operation that will traverse all txn in txn pool unless a node is the block proposer, so the number of txn in pool is not the bottleneck at all. Instead, RAM usage is the current problem